internal/testprovider/schema_large_tokens.go

```diff
@@ -0,0 +1,997 @@
+// Copyright 2016-2024, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testprovider
+
+import (
+	"github.com/YakDriver/regexache"
+	"github.com/aws/aws-sdk-go/service/quicksight"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ProviderLargeTokens() *schema.Provider {
+	resourceTemplate := func() *schema.Resource {
+		return &schema.Resource{
+			Schema:      resourceQuicksightTemplate(),
+			Description: "Quicksight template",
+		}
+	}
+
+	return &schema.Provider{
+		Schema: map[string]*schema.Schema{},
+		ResourcesMap: map[string]*schema.Resource{
+			"aws_quicksight_template": resourceTemplate(),
+		},
+	}
+}
+
+func resourceQuicksightTemplate() map[string]*schema.Schema {
```

Comment:
I pulled this from the actual `aws_quicksight_template` resource for completeness sake. It's not the entire schema (i've commented out some parts so we could add them later)